app/src/main/java/com/example/keybindhelper/RecyclerViewAdapters/GroupAdapter.java

```diff
@@ -68,7 +68,7 @@ public void onBindViewHolder(@NonNull GroupViewHolder holder, int position) {
                     null
             );
             pd.validation= text -> new ValidatorResponse(
-                    text.equals(group.name) || CurrentProjectManager.CurrentProject.isGroupNameAvailable(text),
+                    text.equals(group.name.toString()) || CurrentProjectManager.CurrentProject.isGroupNameAvailable(text),
```

Comment:
Can't place "equal" between two objects of inconvertible types. Changed group to String.

Reply:
This shouldn't be needed, as "name" is already declared as a String:

    public Group(String name, long projectID){